lib/matchers/ip.js

```diff
@@ -0,0 +1,23 @@
+var factory  = require('../factory');
+
+var IP_REGEX = /^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])$/;
+
+module.exports = factory({
+  initialize: function(opts) {
+    if (!opts || opts.version !== 4) {
+      throw new Error("Must be initialized with version 4");
```

Comment:
We spoke about it. The problem is, it would break compatibility if we added IPv6 support, and we wanted to make `ip` the generic checker, consumers who expected `ip` to just do IPv4 would suddenly start have different behaviour. Ideally, we would need to upgrade major version.